test/e2e/framework/resource_usage_gatherer.go

```diff
@@ -603,21 +603,21 @@ type kubemarkResourceUsage struct {
 	CPUUsageInCores         float64
 }
 
-func getMasterUsageByPrefix(prefix string) (string, error) {
+func getControlPlaneUsageByPrefix(prefix string) (string, error) {
 	sshResult, err := e2essh.SSH(fmt.Sprintf("ps ax -o %%cpu,rss,command | tail -n +2 | grep %v | sed 's/\\s+/ /g'", prefix), APIAddress()+":22", TestContext.Provider)
 	if err != nil {
 		return "", err
 	}
 	return sshResult.Stdout, nil
 }
 
-// getKubemarkMasterComponentsResourceUsage returns the resource usage of kubemark which contains multiple combinations of cpu and memory usage for each pod name.
-func getKubemarkMasterComponentsResourceUsage() map[string]*kubemarkResourceUsage {
+// getKubemarkControlPlaneComponentsResourceUsage returns the resource usage of kubemark which contains multiple combinations of cpu and memory usage for each pod name.
+func getKubemarkControlPlaneComponentsResourceUsage() map[string]*kubemarkResourceUsage {
 	result := make(map[string]*kubemarkResourceUsage)
 	// Get kubernetes component resource usage
-	sshResult, err := getMasterUsageByPrefix("kube")
+	sshResult, err := getControlPlaneUsageByPrefix("kube")
 	if err != nil {
-		Logf("Error when trying to SSH to master machine. Skipping probe. %v", err)
+		Logf("Error when trying to SSH to controlplane machine. Skipping probe. %v", err)
```

Comment:
tiny non-blocking nit: when it's not a variable name we should leave a space between the words
```suggestion
		Logf("Error when trying to SSH to control plane machine. Skipping probe. %v", err)
```